datalake/processing_stack.py

```diff
@@ -33,6 +35,14 @@ def __init__(
             self, "processing_assets", deploy_env=deploy_env, application_layer=application_layer
         )
 
+        aws_ssm.StringParameter(
+            self,
+            ResourceName.ProcessingAssetsTableName.value,
+            parameter_name=f"/{deploy_env}/{ResourceName.ProcessingAssetsTableName.value}",
+            description="Name of the Processing Assets Table",
```

Comment:
Considering the parameter name, does this description add anything? Ditto for the other parameters.